app/javascript/react/components/RestaurantShowContainer.js

```diff
@@ -3,21 +3,27 @@ import { Redirect } from 'react-router-dom'
 
 import RestaurantShowTile from './RestaurantShowTile'
 
-const RestaurantShowContainer = (props) => {  
+const RestaurantShowContainer = (props) => {
   const [restaurant, setRestaurant] = useState({})
-  const [currentUser, setCurrentUser ] = useState({})
-  const [ redirect, shouldRedirect ] = useState(false)
+  const [reviews, setReviews] = useState([])
+  const [currentUser, setCurrentUser] = useState({})
+  const [redirect, shouldRedirect] = useState(false)
 
   useEffect(() => {
+    fetch_restaurant_data()
+    fetch_reviews_for_restaurant()
```

Comment:
Updated the code so that we are just doing one fetch for the restaurant information, which returns the review data as well. With the update, we also removed the API Controller for Reviews and the namespace for the Review API in routes.rb. This also removed the controller test for Reviews API.

We created a new private method in the Restaurants API controller so that it would grab the serialized Reviews data